.github/workflows/continuous-integration.yml

```diff
@@ -73,6 +69,78 @@ jobs:
           name: "phpunit-sqlite-${{ matrix.php-version }}-${{ matrix.dbal-version }}-coverage"
           path: "coverage*.xml"
 
+  phpunit-oci8:
+    name: "PHPUnit on OCI8"
+    runs-on: "ubuntu-20.04"
+    needs: "phpunit-smoke-check"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.4"
+        oracle-version:
+          - "11"
+        dbal-version:
+          - "3@dev"
```

Comment:
Are we waiting for a release of doctrine/dbal#5466 here? Or is there anything else that prevents us from using a stable DBAL for the tests?

Reply:
I think we need to wait until https://github.com/doctrine/dbal/pull/5268 is released.

Reply:
@FaritSlv it has been released :)

Reply:
Ok, need to update branch.

How do I change pr to 2.13?

Reply:
Looks like you found by yourself :)